dpt/Cargo.toml

```diff
@@ -8,18 +8,18 @@ repository = "https://github.com/sorpaas/rust-devp2p"
 edition = "2018"
 
 [dependencies]
-secp256k1-plus = "0.5"
-rand = "0.3.12"
-sha3 = "0.6"
 etcommon-bigint = "0.2"
 etcommon-hexutil = "0.2"
 etcommon-rlp = "0.2"
 futures = "0.1"
-tokio-io = "0.1"
+log = "0.3"
+rand = "0.3.12"
+secp256k1-plus = "0.5"
```

Comment:
We kind of want to get rid of this in the future. `secp256k1-plus` is simply `rust-secp256k1` with `lazy_static`.